library/src/main/java/com/pokegoapi/api/listener/SettingsListener.java

```diff
@@ -0,0 +1,19 @@
+package com.pokegoapi.api.listener;
+
+import com.pokegoapi.api.PokemonGo;
+
+/**
+ * Created by RebliNk17 on 09/04/2017.
+ */
+public interface SettingsListener extends Listener {
+
+	/**
+	 * Called when a new API is forced
+	 *
+	 * @param go the current api
+	 * @param currentVersion the current supported api version
+	 * @param forcedVersion the new forced api.
+	 */
+	 void onNewVersionForced(PokemonGo go, String currentVersion, String forcedVersion);
```

Comment:
[ant:checkstyle] [WARN] /home/jasper/Documents/PokemonGo/PokeGOAPI-Java/library/src/main/java/com/pokegoapi/api/listener/SettingsListener.java:17: 'method def modifier' have incorrect indentation level 3, expected level should be 2. [Indentation]